File: First assignment/Python/1st question.py
count = 0
add = 0
max_no = int(input("Please enter the max integer(< or = to 100) till you want to add = "))
while max_no > 100:
    max_no = int(input("The integer have to be less than 100 = "))
while count < max_no:
    count = count+1
    add = add+count
print("The sum of integers till", max_no, "=", add)

File: First assignment/C/1st question.c

#include <stdio.h>

int main()
{
    int count = 0;
	int add = 0;
	int max_no;
	printf("Please enter the max integer(< or = to 100) till you want to add = ");
	scanf("%d", &max_no);
	while (max_no > 100){
        printf("The integer have to be less than 100 = ");
        scanf("%d", &max_no);
	}
	while(count < max_no){
	    count = count + 1;
	    add = add + count;
	}
	printf("The sum of integers till %d = %d", max_no, add);

    return 0;
}
